lsp/src/main/java/org/aya/lsp/server/AyaLanguageServer.java

```diff
@@ -376,6 +405,26 @@ public ComputeTermResult computeTerm(@NotNull ComputeTermResult.Params input, Co
     return primFactories.getOrPut(owner.underlyingLibrary(), LspPrimFactory::new);
   }
 
+  @LspRequest("options/updateOptions")
+  public void updateOptions(@NotNull ServerOptions options) {
```

Comment:
Prefix the method name with `aya/` in case LSP spec adds some methods prefixed with `options`

Reply:
> `aya/options/updateOptions` or `aya/updateOptions` ?

Choose the one you like most 😉